libs/Microsoft.MixedReality.WebRTC/DataChannel.cs

```diff
@@ -58,6 +58,22 @@ public enum ChannelState
             Closed = 3
         }
 
+        /// <summary>
+        /// Type of message sent or received through the data channel.
+        /// </summary>
+        public enum MessageKind
+        {
+            /// <summary>
+            /// The message is a binary message.
+            /// </summary>
+            Binary = 1,
+
+            /// <summary>
+            /// The message is a text message.
+            /// </summary>
+            Text = 2
```

Comment:
Should specify what "text message" means (UTF-8 string).

Reply:
I've updated the summary, see 70af869ce60cc2bda6068c0abfc4af5fded8c348